Cognite.Simulator.Utils/RoutineLibraryBase.cs

```diff
@@ -215,12 +241,21 @@ private async Task ReadRoutineRevisions(CancellationToken token)
 
             var routineRevisions = await routineRevisionsRes.ToListAsync(token).ConfigureAwait(false);
 
-            foreach (var routineRev in routineRevisions)
+            if (routineRevisions.Any())
             {
-                if (!RoutineRevisions.ContainsKey(routineRev.Id.ToString()))
+                foreach (var routineRev in routineRevisions)
                 {
                     ReadAndSaveRoutineRevision(routineRev);
                 }
+
+                var maxCreatedMs = RoutineRevisions
```

Comment:
nitpick : can we rename the variables to something like ? 

`maxCreatedMs` -> `maxCreatedTimestamp` 

`maxCreatedDt ` -> `maxCreatedDateTime`

Reply:
OKI